pages.de/common/chmod.md

```diff
@@ -29,3 +29,7 @@
 - Ändert Genehmigungen rekursiv, indem Sie [g]ruppe und Anderen ([o]thers) die Erlaubniss zum schreiben ([w]rite) geben:
 
 `chmod -R g+w,o+w {{verzeichnis}}`
+
+- Kurzschreibweise um Rechte für User, Groups und others zu legen indem die Berechtigungen in Oktal angelegt werden:
```

Comment:
The translation should be made from the primary (English) pages.

If you want to add an extra example you should send a PR for edit the `pages/common/chmod.md`, and you can include the update the German translation for `pages.de/common/chmod.md` in that PR.